### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

**Review by matechris — APPROVED** [review]
Great job!

File: app/cafe.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
```
Comment (Nattalli) [inline app/cafe.py:15]: I think that you may make more informative error message

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Comment by HStakhiv** [comment]
Is it better now?

**Review by Nattalli — APPROVED** [review]
Looks good 😄

**Pull request closed without merging**